### PR TITLE
fix dead link in `README`

### PR DESCRIPTION
**Pull request by moritzwilksch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,5 +83,5 @@ result = table.search([100, 100]).limit(2).to_pandas()
 ```
 
 ## Blogs, Tutorials & Videos
-* 📈 <a href="https://blog.eto.ai/benchmarking-random-access-in-lance-ed690757a826">2000x better performance with Lance over Parquet</a>
+* 📈 <a href="https://blog.lancedb.com/announcing-lancedb-5cb0deaa46ee-2/">2000x better performance with Lance over Parquet</a>
```
Comment (eddyxu) [inline README.md:86]: Should we actually use this one? https://blog.lancedb.com/announcing-lancedb-5cb0deaa46ee-2/ ?

Comment (moritzwilksch) [reply README.md:86]: that link is the same as in the diff, no? Or are you considering linking an entirely different page?

Comment (eddyxu) [reply README.md:86]: Thanks @moritzwilksch could you rebase this PR?

It should be good to go

Comment (moritzwilksch) [reply README.md]: the new link also seems dead now

**Pull request closed without merging**